plugin.xml

```diff
@@ -16,7 +16,7 @@ xmlns:android="http://schemas.android.com/apk/res/android"
         <engine name="cordova-ios" version=">=5.1.1"/>
     </engines>
 
-    <dependency id="cordova-plugin-firebase-analytics" url="https://github.com/OutSystems/cordova-plugin-firebase-analytics.git#outsystems"/>
+    <dependency id="cordova-plugin-firebase-analytics" url="https://github.com/OutSystems/cordova-plugin-firebase-analytics.git#fix/remove-hook-unzipAndCopy"/>
```

Comment:
Shouldn't we point it to a specific version of the plugin?

Reply:
@alexgerardojacinto I actually have a question here. This should point to a release TAG or if there is none to outsystems as it was before. But pointing to a fix branch I don't think its a good solution :D

Reply:
This is outdated @GFOutsystems , I pointed to the outsystems branch in the next commit, because for now we do not have a release tag :)